app/forms/poor_performance_form.rb

```diff
@@ -0,0 +1,22 @@
+class PoorPerformanceForm < Form
```

Comment:
What do you think about namespacing this form under `Journeys:AdditionalPaymentsForTeaching`?

Reply:
I was thinking maybe of a system similar to the way the views are loaded; first it looks for the view in the relevant journey folder and then it looks in the shared `claims` folder.